anvil_consortium_manager/apps.py

```diff
@@ -1,9 +1,24 @@
 from django.apps import AppConfig
+from django.conf import settings
+from django.utils.module_loading import import_string
 
 
-class AnVILonsortiumManagerConfig(AppConfig):
+class AnVILConsortiumManagerConfig(AppConfig):
     """Configuration for the anvil_consortium_manager app."""
 
     default_auto_field = "django.db.models.BigAutoField"
     name = "anvil_consortium_manager"
     verbose_name = "AnVIL Consortium Manager"
+
+    def ready(self):
+        super().ready()
+        # Register specified adapters.
+        # Import here because importing outside of this method raises the AppRegistryNotReady exception.
+        from anvil_consortium_manager.adapters.workspace import (
+            workspace_adapter_registry,
+        )
+
+        adapter_modules = settings.ANVIL_WORKSPACE_ADAPTERS
```

Comment:
Is it okay to have zero adapters? If not, you may want to validate the list is not empty.

Reply:
Good idea, I added a check.